tests/e2e/runAnalysis.spec.ts

```diff
@@ -54,6 +54,11 @@ test("Can request a scenario analysis run", async ({ page, baseURL, headless })
 
   await expect(page.locator("#prevalence-container")).toBeVisible();
   await page.locator("#prevalence-container").scrollIntoViewIfNeeded();
+  await expect(page.getByText("New per day").first()).toBeVisible();
+  await page.locator("#infectionsDailySwitch").check();
+  await expect(page.getByRole("button", { name: "New infections Number of new" })).toBeVisible();
+  await page.locator("#infectionsDailySwitch").setChecked(false);
+  await expect(page.getByRole("button", { name: "Prevalence Number of" })).toBeVisible();
```

Comment:
I think just for readability it would be good to comment this section.. and move it down to after line 65 so it isn't interrupting the sequence of static checks on the initial prevalence view. 
```suggestion
  // Check can toggle time series to "New per day" and back
  await expect(page.getByText("New per day").first()).toBeVisible();
  await page.locator("#infectionsDailySwitch").check();
  await expect(page.getByRole("button", { name: "New infections Number of new" })).toBeVisible();
  await page.locator("#infectionsDailySwitch").setChecked(false);
  await expect(page.getByRole("button", { name: "Prevalence Number of" })).toBeVisible();
```